src/common/image.c

```diff
@@ -771,17 +771,18 @@ int32_t dt_image_duplicate_with_version(const int32_t imgid, const int32_t newve
      "   raw_auto_bright_threshold, raw_black, raw_maximum,"
      "   caption, description, license, sha1sum, orientation, histogram, lightmap,"
      "   longitude, latitude, altitude, color_matrix, colorspace, version, max_version, history_end,"
-     "   position, aspect_ratio)"
+     "   position, aspect_ratio, import_timestamp)"
      " SELECT NULL, group_id, film_id, width, height, filename, maker, model, lens,"
      "       exposure, aperture, iso, focal_length, focus_distance, datetime_taken,"
      "       flags, width, height, crop, raw_parameters, raw_denoise_threshold,"
      "       raw_auto_bright_threshold, raw_black, raw_maximum,"
      "       caption, description, license, sha1sum, orientation, histogram, lightmap,"
-     "       longitude, latitude, altitude, color_matrix, colorspace, NULL, NULL, 0, ?1, aspect_ratio"
+     "       longitude, latitude, altitude, color_matrix, colorspace, NULL, NULL, 0, ?1, aspect_ratio, ?3"
      " FROM main.images WHERE id = ?2",
      -1, &stmt, NULL);
   DT_DEBUG_SQLITE3_BIND_INT64(stmt, 1, new_image_position);
   DT_DEBUG_SQLITE3_BIND_INT(stmt, 2, imgid);
+  DT_DEBUG_SQLITE3_BIND_INT(stmt, 3, time(0));
```

Comment:
And maybe it is better to do that in SQL directly anyway. For consistency reasons, look at what is done for write_timestamp:

`SET write_timestamp = STRFTIME('%s', 'now')`

Reply:
I read some time ago that the system routines have been modified to be able to pass this fateful date without any problem (time_t has been enlarged to 64 bits).
See https://en.wikipedia.org/wiki/Year_2038_problem.
That's why I opted for this representation which has several advantages: 
- Less space occupied in the comic book.
- Less load since there are less conversions to do.
- A simplified coding, always thanks to the conversions.

Nevertheless to ensure homogeneity I could change to "strftime" values.

Reply:
... Using a signed 64-bit value introduces a new wraparound date that is over twenty times greater than the estimated age of the universe: approximately 292 billion years from now, at 15:30:08 UTC on Sunday, 4 December 292,277,026,596. (from Wikipedia) ;-)

Reply:
All good for the year 2038 ! I'd still prefer having strftime here to have the same code on all platforms as time() is an OS call and strftime() is already in use in other places.